docker/generic/start_proxy.py

```diff
@@ -1515,6 +1519,21 @@ def start_envoy(args):
     t.start()
     return proc
 
+def sigterm_handler(signum, frame):
+    """ Handler for SIGTERM and SIGINT, pass the SIGTERM to all child processes. """
+    signame = signal.Signals(signum).name
+    logging.warning("{}: got signal: {}".format(datetime.utcnow().isoformat(timespec='microseconds'), signame))
```

Comment:
nit: change loging.basicConfig on line 1539 to log timestamps

Reply:
Done